modal-go/sandbox_filesystem.go

```diff
@@ -0,0 +1,145 @@
+package modal
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	pb "github.com/modal-labs/libmodal/modal-go/proto/modal_proto"
+)
+
+// SandboxFile represents an open file in the sandbox filesystem.
+// It implements io.Reader, io.Writer, io.Seeker, and io.Closer interfaces.
+type SandboxFile struct {
+	fileDescriptor string
+	taskId         string
+	ctx            context.Context
+}
+
+// newFile creates a new File object.
+func newSandboxFile(ctx context.Context, fileDescriptor, taskId string) *SandboxFile {
```

Comment:
Typo in the comment here. But also you probably don't need this for Go since it's only called internally, might actually be clearer to construct it inline in `Open()` with code like `&SandboxFile{ ... }` and then you just delete this method.